static/docs/commands-reference/push.md

```diff
@@ -1,7 +1,7 @@
 # push
 
 Uploads files and directories under DVC control to the
-[remote storage]('doc/commands-reference/remote').
```

Comment:
This one seemed like an error, I changed it (along with a few more occurrences of similar cases below)